roles/monitor_host/tasks/main.yml

```diff
@@ -9,7 +9,7 @@
     return_content: True
   register: cluster
   delegate_to: bastion
-  until: cluster is successful
+  until: cluster_hosts is successful
```

Comment:
According to the test last execution in DCI, it seems the 'cluster_hosts' variable is not correct.
The logs are saying:
```
"The conditional check 'cluster_hosts is successful' failed. The error was: The 'failed' test expects a dictionary"
```
Should the variable be simply named 'cluster' instead of 'cluster_hosts' here?

Reply:
This is resolved. Thanks